### PR TITLE
Remove AdGuard DNS filter from the FiltersRegistry

### PR DESCRIPTION
**Pull request by zloyden**
https://github.com/AdguardTeam/FiltersRegistry/issues/800

### CODE REVIEW

File: filters/filter_15_DnsFilter/metadata.json [hunk 1]
```diff
@@ -20,7 +20,9 @@
     "reference:118",
     "reference:102",
     "reference:13",
-    "reference:16"
+    "reference:16",
+    "obsolete"
   ],
-  "trustLevel": "full"
+  "trustLevel": "full",
+  "disabled": true
```
Comment (ameshkov) [inline filters/filter_15_DnsFilter/metadata.json:27]: Not sure we should make it `disabled` though, let it continue updating for some time.

File: filters/filter_15_DnsFilter/metadata.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "filterId": 15,
-  "name": "AdGuard DNS filter",
+  "name": "(Obsolete) AdGuard DNS filter",
```
Comment (ameshkov) [inline filters/filter_15_DnsFilter/metadata.json:3]: The list itself is not obsolete so I suggest not changing the name

Comment (zloyden) [reply filters/filter_15_DnsFilter/metadata.json]: Done.

File: filters/filter_15_DnsFilter/metadata.json (hunk 1 above)
Comment (zloyden) [reply filters/filter_15_DnsFilter/metadata.json]: So, we can add it later?
I left only the tag "obsolete" as you suggested earlier.

**Review by Alex-302 — APPROVED** [review]

**Pull request closed without merging**